src/util/languageUtils.test.js

```diff
@@ -60,6 +61,13 @@ describe('getBrowserLang', () => {
 		const lang = getBrowserLang(request, supportedLangs);
 		expect(lang).toBe(false);
 	});
+	it('returns en-US instead of en-AU for "en" if en-US is preferred', () => {
```

Comment:
we should make sure that this respects the priority of languages as provided by the browser. I believe this is why my browser was working with most sites but failing with mup-web - https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.4